modules/core/src/main/scala/iota/internal/TypeListMacros.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016-2017 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package iota
+package internal
+
+import scala.reflect.macros.blackbox.Context
+
+private[iota] final class TypeListMacros(cc: Context)
```

Comment:
Weird alignment I think, but I'm not sure if this is your standard scalaFMT and I just have different habits. I would expect:

```scala
class A(
 ..args
) extends Bla {
}
```